functions/Install-DbaWhoIsActive.ps1

```diff
@@ -79,27 +79,38 @@ function Install-DbaWhoIsActive {
 	)
 	
 	begin {
+        $DbatoolsData = Get-DbaConfigValue -Name "Path.DbatoolsData"
 		$temp = ([System.IO.Path]::GetTempPath()).TrimEnd("\")
 		$zipfile = "$temp\spwhoisactive.zip"
 
 		if ($LocalFile -eq $null -or $LocalFile.Length -eq 0) {
 			$baseUrl = "http://whoisactive.com/downloads"
 			$latest = ((Invoke-WebRequest -uri http://whoisactive.com/downloads).Links | where-object {$PSItem.href -match "who_is_active"} | Select-Object href -First 1).href	
-			if ($PSCmdlet.ShouldProcess($env:computername, "Downloading sp_WhoisActive")) {
-				try {
-					Write-Message -Level Verbose -Message "Downloading sp_WhoisActive zip file, unzipping and installing."
-					$url = $baseUrl + "/" + $latest
+			$LocalCachedCopy = Join-Path -Path $DbatoolsData -ChildPath $latest;
+
+			if (Test-Path -Path $LocalCachedCopy -PathType Leaf) {
+				Write-Message -Level Verbose -Message "Locally-cached copy exists, not downloading."
```

Comment:
Would reword this to say maybe 

> Locally-cached copy found, skipping download

Most of the commands we use "skipping" instead of "not".